qiskit/circuit/quantumcircuit.py

```diff
@@ -4683,6 +4685,202 @@ def mcp(
             copy=False,
         )
 
+    def mcrx(
+        self,
+        theta: ParameterValueType,
+        q_controls: Union[QuantumRegister, List[Qubit]],
```

Comment:
It would be nice to use the native typehints we started using since Python 3.7 is the minimal support version, i.e.
```suggestion
        q_controls: QuantumRegister | list[Qubit],
```